configure

```diff
@@ -81,6 +81,9 @@ fi
 if echo ${etcdir} | grep -qs /usr; then
     etcdir=$(echo ${etcdir} | sed -e "s^/usr^^")
 fi
+if [ -n "$sysconfdir" ]; then
+    etcdir=$sysconfdir
+fi
```

Comment:
OK. By the way you may want to revisit the `etcdir` fixup because `/usr/local/etc` is actually the correct place for locally installed programs.

http://man7.org/linux/man-pages/man7/hier.7.html